shelf/hook/null_manager.py

```diff
@@ -0,0 +1,20 @@
+class NullManager(object):
```

Comment:
Python doesn't do interfaces as I understand it. I could create a class that throws exceptions for both functions and then override with a class that doesn't nothing but it feels a bit tedious. What I should really be doing is duck typing in the places I am using it. However, my contract with the container is that I will provide an object that has those two functions.

There is also a concept of an Abstract Base Class added in 2.6 (not sure if 2.6.6 though). It would make it so that at least you can't instantiate it.

Thoughts? Which part of this makes it a problem? The doc blocks not pointing to a definition that has both those methods?